src/providers/providers/index.ts

```diff
@@ -146,12 +146,26 @@ export const BINANCECHAINWALLET: IProviderInfo = {
   check: "isBinanceChainWallet"
 };
 
+/**
+ * @deprecated Use CoinbaseWalletSdk
+ */
 export const WALLETLINK: IProviderInfo = {
   id: "walletlink",
   name: "Coinbase Wallet",
-  logo: WalletLinkLogo,
+  logo: CoinbaseWalletLogo,
   type: "qrcode",
-  check: "isWalletLink",
+  check: "isCoinbaseQr",
```

Comment:
i believe it needs to be
```suggestion
  check: "isCoinbaseWallet",
```